docker-compose.yml

```diff
@@ -6,7 +6,9 @@ services:
       - ./back-end:/visual-programming/back-end
     ports:
       - "8000:8000"
-    command: bash -c "cd vp && pipenv run python manage.py runserver 0.0.0.0:8000"
```

Comment:
On my machine at least I need this extra `cd vp` part of the command. I think this is because the `docker-compose.yml` donesn't the have same/similar line that the individual `Dockerfile` has of `WORKDIR /..../vp` to change it. 

Reverting back to `cd vp && pipenv ...` still runs for me with the other changes you made.